Microsoft.Recognizers.Text.DateTime/Extractors/BaseDatePeriodExtractor.cs

```diff
@@ -102,6 +103,40 @@ private List<Token> MergeTwoTimePoints(string text)
             return ret;
         }
 
+        private List<Token> SingleTimePointWithPatterns(string text)
```

Comment:
Can you add a comment in the code giving an example of the cases that would be covered here?